cli/src/account_completion.rs

```diff
@@ -0,0 +1,62 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use dialoguer::Completion;
+
+pub(crate) struct AccountCompletion<'a> {
+    options: [&'a str; 33],
+}
+
+pub(crate) const ACCOUNT_COMPLETION: AccountCompletion = AccountCompletion {
+    options: [
+        "addresses",
+        "balance",
+        "burn-native-token",
+        "burn-nft",
+        "claim",
+        "consolidate",
+        "create-alias-output",
+        "decrease-native-token-supply",
+        "destroy-alias",
+        "destroy-foundry",
+        "exit",
+        "faucet",
+        "increase-native-token-supply",
+        "mint-native-token",
+        "mint-nft",
+        "new-address",
+        "output",
+        "outputs",
+        "send",
+        "send-micro",
+        "send-native-token",
+        "send-nft",
+        "sync",
+        "transactions",
+        "unspent-outputs",
+        "vote",
+        "stop-participating",
+        "participation-overview",
+        "voting-power",
+        "increase-voting-power",
+        "decrease-voting-power",
+        "voting-output",
+        "help",
+    ],
+};
+
+impl<'a> Completion for AccountCompletion<'a> {
+    fn get(&self, input: &str) -> Option<String> {
+        let matches = self
+            .options
+            .iter()
+            .filter(|option| option.starts_with(input))
```

Comment:
maybe we want to switch this to contains? just typing power, would be nice that all options show up

Reply:
Yeah but you have to return only one String, so what do you return when you provide power ?

Reply:
ah i see, thats silly. never mind then